src/styles/index.css

```diff
@@ -0,0 +1,98 @@
+@import "./reset.css";
+@import "./stars.css";
+
+:root {
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  box-sizing: border-box;
+  width: 200px;
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  display: block;
+}
+
+.card__title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin: 0 0 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  color: #616070;
+  margin-bottom: 16px;
+  line-height: 14px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+  line-height: 14px;
+}
+
+.card__name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__number {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 24px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 16px;
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background: #00acdc;
+  color: white;
+  text-decoration: none;
+  border-radius: 5px;
+  border: 1px solid #00acdc;
+  box-sizing: border-box;
+
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+}
+
+.card__buy:hover {
+  background: transparent;
+  color: #00acdc;
+  line-height: 16px;
+  text-align: center;
+  transition: 0.3s;
```

Comment:
If you add transition on hover, it doesn’t work correctly (your styles add smoothly but remove quickly) - you need add transition for element (not on hover)